utils/clean-up-job.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: buildkite-agent-stack-k8s-job-cleanup
+  namespace: buildkite
```

Comment:
Further to @zhming0's comment above, given that we make users specify a namespace on install, we should remove the namespace here and instead instruct them to "pass `-n buildkite` (or `-n your-namespace`) to `kubectl apply`" (or similar)

```suggestion
```